allennlp_models/rc/dataset_readers/transformer_squad.py

```diff
@@ -222,9 +257,10 @@ def text_to_instance(
         tokenized_question: List[Token],
         context: str,
         tokenized_context: List[Token],
-        answers: List[str],
-        token_answer_span: Optional[Tuple[int, int]],
+        answers: List[str] = None,
+        token_answer_span: Optional[Tuple[int, int]] = None,
         additional_metadata: Dict[str, Any] = None,
+        for_training: bool = False,
```

Comment:
I'd rather have flag names that explain what they do, instead of referring to the context to set them in.

For example, this flag could be called `always_add_answer_span`.